modules/rn_3_70.adoc

```diff
@@ -1,97 +1,69 @@
 
-// PROJQUAY-465 
-=== Proxy
+Known issues:
 
-* link:https://issues.redhat.com/browse/PROJQUAY-465[PROJQUAY-465]. Quay as a cache proxy / pull-through cache for other registries
- 
-==== Design 
+* link:https://issues.redhat.com/browse/PROJQUAY-3590[PROJQUAY-3590]. Quay 3.7.0 pull from cache should return quota exceeded error rather than general 403 error code.
 
-* link:https://issues.redhat.com/browse/PROJQUAY-2888[PROJQUAY-2888]. Quay as a cache proxy - Design: Proxy org creation and configuration
+Fixed:
 
-* link:https://issues.redhat.com/browse/PROJQUAY-2889[PROJQUAY-2889]. Quay as a cache proxy - Design: Upstream image pull flow
+=== {productname} feature tracker
 
-* link:https://issues.redhat.com/browse/PROJQUAY-2890[PROJQUAY-2890]. Quay as a cache proxy - Design: Local storage of upstream image layers and manifests
+New features have been added to {productname}, some of which are currently in Technology Preview. Technology Preview features are experimental features and are not intended for production use.
 
-* link:https://issues.redhat.com/browse/PROJQUAY-2891[PROJQUAY-2891]. Quay as a cache proxy - Design: Quota Management in Cache Proxy org	Spike
+Some features available in previous releases have been deprecated or removed. Deprecated functionality is still included in {productname}, but is planned for removal in a future release and is not recommended for new deployments. For the most recent list of major functionality deprecated and remove with {productname} 3.7, refer to the table below. Additional details for more fine-grained functionality that has been deprecated and removed are listed after the table.
 
-==== User stories
+=== quay-operator
 
-* link:https://issues.redhat.com/browse/PROJQUAY-3029[PROJQUAY-3029]. As a Quay user, I want to create and configure pull-through proxy orgs via UI
+Added/Changed:
 
-* link:https://issues.redhat.com/browse/PROJQUAY-3030[PROJQUAY-3030]. As a Quay user I want to be able to proxy images through Quay orgs
+* Advanced Clair configuration is now available for {productname} 3.7. The following features are now available to Quay administrators on the Quay Operator:
 
-* link:https://issues.redhat.com/browse/PROJQUAY-3033[PROJQUAY-3033]. As a Quay user I want proxied images to be stored in Quay so that my pulls are faster
+** Configuration of Clair's updater set through the Quay Operator.
+** Configuration of the database connection string through the Quay Operator.
+** Configuration of custom certificates into the Clair deployment, which allows support of internal HTTPS proxies.
+** Override, or re-size, the size and storage class of the Postgres Database PVC.
+** Support for alternative fully qualified domain names (FQDN) for Clair that can leverage a global load balancing mechanism fronting different clusters running Clair.
```

Comment:
This is misleading or I did not understand it. There is no global load balancing for Clair.

Reply:
@ricardomaraschini should it be removed, or say

** Support for alternative fully qualified domain names (FQDN) for Clair that users can leverage a global load balancing mechanism fronting different clusters running Clair.

Reply:
Removed. There is no global load balancing support for Clair.